ddl/ddl_api.go

```diff
@@ -74,7 +74,10 @@ const (
 	longBlobMaxLength     = 4294967295
 	// When setting the placement policy with "PLACEMENT POLICY `default`",
 	// it means to remove placement policy from the specified object.
-	defaultPlacementPolicyName = "default"
+	defaultPlacementPolicyName        = "default"
+	tiflashCheckPendingTablesWaitTime = 2500 * time.Millisecond
+	tiflashCheckPendingTablesTick     = 100
```

Comment:
Could we rename this variable? like `tiflashBatchPendingTablesLimit`. I do not understand the meaning of this tick.